### ISSUE TITLE
Pekko support

### ISSUE THREAD

**Issue by SamTheisens** [ReactiveMongo/ReactiveMongo#1244]
### Feature request

[Pekko](https://pekko.apache.org/what-is-pekko.html), the OSS fork of Akka has reached 1.0.0. Many projects will want to move away from Akka since their introduction of the Business Source License model for Akka version 2.7 and above.
A pekko based distribution would allow people to keep using ReactiveMongo.

**Comment by Fristi** [comment]
@cchantep Sam did already great work adding Pekko support, as suggested in the PR would it be an idea to finish that PR _without_ Scala 3.3 support and make a release?

- From my experience not a lot of people have gone migrating to Scala 3.x in general for various reasons (libraries, tooling, etc)
- I think it's also likely that existing code bases are stuck on other dependencies before moving to Scala 3.x anyway  (industry is stuck on macro dependencies)
- Some users might drop ReactiveMongo if the progress is staling too much at some point

In short, it would help us a great deal to have this being release for Scala 2.x ❤️
